css/css-contain/content-visibility/content-visibility-080.html

```diff
@@ -21,6 +21,7 @@
 <script>
 test(() => {
   const range = document.caretRangeFromPoint();
+  html.style.contentVisibility = "visible";
```

Comment:
If it's purely about the output let's use `t.add_cleanup` to remove the `style` element so we can be sure it doesn't affect the test.